gpt4all-bindings/python/docs/gpt4all_faq.md

```diff
@@ -36,4 +36,8 @@ In newer versions of llama.cpp, there has been some added support for NVIDIA GPU
 
 1. Check to make sure the huggingface model is available in one of our three supported architectures
 2. If it is, then you can use the conversion script inside of our pinned llama.cpp submodule for GPTJ and LLAMA based models
-3. Or if your model is an MPT model you can use the conversion script located directly in this backend directory under the scripts subdirectory 
\ No newline at end of file
+3. Or if your model is an MPT model you can use the conversion script located directly in this backend directory under the scripts subdirectory 
+
+## What is the best way to write prompt templates?
+
+Prompt Engineering is an evolving field with many new discoveries in how to navigate a model's latent space. Some commands that work on a model may not work on another. Please [join the discussion](https://github.com/nomic-ai/gpt4all/issues/631) on our github for more information on how to write good prompt templates for different models.
```

Comment:
I don't think we want our FAQ/WIKI linking to an issue. I think an issue is the wrong place for that discussion as it seems open ended and without a clear ... issue ;)

Reply:
Thanks for the feedback! I'll update it soon with a few template examples to guide users.

Reply:
I think a section under the Wiki tab in the docs on prompt engineering may be the solution here